tests/tensor/rewriting/test_uncanonicalize.py

```diff
@@ -23,26 +22,26 @@
 from tests.link.test_link import make_function
 
 
-class TestMaxAndArgmax:
-    def test_optimization(self):
-        # If we use only the max output, we should replace this op with
-        # a faster one.
-        mode = pytensor.compile.mode.get_default_mode().including(
-            "canonicalize", "fast_run"
-        )
+# class TestMaxAndArgmax:
+#     def test_optimization(self):
+#         # If we use only the max output, we should replace this op with
+#         # a faster one.
+#         mode = pytensor.compile.mode.get_default_mode().including(
+#             "canonicalize", "fast_run"
+#         )
 
-        for axis in [0, 1, -1]:
-            n = matrix()
+#         for axis in [0, 1, -1]:
+#             n = matrix()
 
-            f = function([n], max_and_argmax(n, axis)[0], mode=mode)
-            topo = f.maker.fgraph.toposort()
-            assert len(topo) == 1
-            assert isinstance(topo[0].op, CAReduce)
+#             f = function([n], max_and_argmax(n, axis)[0], mode=mode)
+#             topo = f.maker.fgraph.toposort()
+#             assert len(topo) == 1
+#             assert isinstance(topo[0].op, CAReduce)
 
-            f = function([n], max_and_argmax(n, axis), mode=mode)
-            topo = f.maker.fgraph.toposort()
-            assert len(topo) == 1
-            assert isinstance(topo[0].op, MaxAndArgmax)
+#             f = function([n], max_and_argmax(n, axis), mode=mode)
+#             topo = f.maker.fgraph.toposort()
+#             assert len(topo) == 1
+#             assert isinstance(topo[0].op, MaxAndArgmax)
 
```

Comment:
tests for ```local_max_and_argmax``` removed.

Reply:
Removed code is equally easy to see in the git diff when reviewing, no need to first comment out?